presto-product-tests/src/main/java/io/prestosql/tests/iceberg/TestIcebergCreateTable.java

```diff
@@ -13,6 +13,8 @@
  */
```

Comment:
nit: I would change commit title to `Make TestIcebergCreateTable retriable`

Reply:
The only reason im doing this is because i added `@Flaky` and it didn't work. So my point is -- fix the retries.
if there was no `@Flaky`, i wouldn't do the change.